server/webapp/WEB-INF/rails.new/spec/controllers/admin/materials/pluggable_scm_controller_spec.rb

```diff
@@ -56,7 +56,7 @@
     end
 
     it "check_connection" do
-      {:post => '/admin/materials/pluggable_scm/check_connection/plugin-id'}.should route_to(:controller => 'admin/materials/pluggable_scm', :action => 'check_connection', :plugin_id => 'plugin-id')
+      {:post => '/admin/materials/pluggable_scm/check_connection/plugin.id-1'}.should route_to(:controller => 'admin/materials/pluggable_scm', :action => 'check_connection', :plugin_id => 'plugin.id-1')
```

Comment:
Line is too long. [202/200]<br>Use the new Ruby 1.9 hash syntax.<br>Space inside { missing.<br>Space inside } missing.